src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/FlowSubflowNamingRuleTest.groovy

```diff
@@ -0,0 +1,73 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.CaseNaming
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+import spock.lang.Specification
+
+class FlowSubflowNamingRuleTest extends Specification {
+
+    private static final String NAMING_APP = 'FlowSubflowNaming'
+
+    private static final String CAMEL_CASE_FILE = 'camelCase.xml'
```

Comment:
Limit to 1 file that contains a flow for each naming case... and check that the correct flow passes in each of the different naming scenarios.

Reply:
done.